controllers/topic.go

```diff
@@ -285,17 +285,19 @@ func (c *ApiController) UploadTopicPic() {
 	memberId := c.GetSessionUsername()
 	fileBase64 := c.Ctx.Request.Form.Get("pic")
 	index := strings.Index(fileBase64, ",")
-	if index < 0 || fileBase64[0:index] != "data:image/png;base64" {
+	picType := fileBase64[11 : index-7]
```

Comment:
Plz change the code to a simple file upload file, use the file content as param instead of base64. Don't do base64 content check any more.